sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -227,36 +218,30 @@ def _build_message(self, body):
     def _set_message_annotations(self, key, value):
         if not self._raw_amqp_message.annotations:
             self._raw_amqp_message.annotations = {}
-
-        if isinstance(self, ServiceBusReceivedMessage):
-            try:
-                del self._raw_amqp_message.annotations[key]
-            except KeyError:
-                pass
-
         if value is None:
             try:
-                del self._raw_amqp_message.annotations[ANNOTATION_SYMBOL_KEY_MAP[key]]
+                del self._raw_amqp_message.annotations[key]
             except KeyError:
                 pass
         else:
-            self._raw_amqp_message.annotations[ANNOTATION_SYMBOL_KEY_MAP[key]] = value
+            self._raw_amqp_message.annotations[key] = value
 
-    def _to_outgoing_message(self):
-        # type: () -> ServiceBusMessage
-        # pylint: disable=protected-access
-        self.message = self.raw_amqp_message._to_outgoing_amqp_message()
+    def _to_outgoing_message(self) -> "ServiceBusMessage":
         return self
 
     @property
-    def raw_amqp_message(self):
-        # type: () -> AmqpAnnotatedMessage
+    def message(self) -> LegacyMessage:
```

Comment:
wondering if the reason there isn't a message setter is because we expect that users probably only used get? and whether, b/c message can't be set anymore, it changes the API?

Reply:
hmm that's a good question....
I mostly assumed that because it was on a ReceivedMessage, it could really only be consumed - but I guess a ReceivedMessage can be resent right?
I think it would be unlikely that someone was building a uamqp message and setting this property, and more likely that they updated the existing message... but could be worth looking into.